File: Commercial/Commercial_Controller/Commercial_Controller/FloorDisplay.cs
using System;
using static System.Threading.Thread;
using static System.Console;

namespace Commercial_Controller
{
    class FloorDisplay
    {
        #region FIELDS
        private readonly Elevator _elevator;
        #endregion

        #region CONSTRUCTOR
        public FloorDisplay(Elevator elevator_)
        {
            _elevator = elevator_;
        }
        #endregion

        #region METHODS
        public void DisplayFloor()
        {
            Sleep(Program.SLEEP_TIME);

            if (_elevator.CurrentFloor > 0)
                WriteLine($"... Elevator {_elevator.ID} of column {_elevator.Column.ID}'s current floor mid-travel: {_elevator.CurrentFloor} ...");

            else if (_elevator.CurrentFloor < 0)
                WriteLine($"... Elevator {_elevator.ID} of column {_elevator.Column.ID}'s current floor mid-travel: B{Math.Abs(_elevator.CurrentFloor)} ...");

        }
        #endregion
    }
}

File: Commercial/Commercial_ControllerJava/src/FloorDisplay.java
import static java.lang.System.out;

public class FloorDisplay
{
    //region FIELDS
    private Elevator elevator;
    //endregion

    //region CONSTRUCTOR
    public FloorDisplay(Elevator elevator)
    {
        this.elevator = elevator;
    }
    //endregion

    //region METHODS
    public void displayFloor()
    {
        Program.sleep();

        if (elevator.getCurrentFloor() > 0)
        {
            out.printf("\n... Elevator %d of column %d's current floor mid-travel: %d ...", elevator.getId(), elevator.getColumn().getId(), elevator.getCurrentFloor());
        }
        else if (elevator.getCurrentFloor() < 0)
            out.printf("\n... Elevator %d of column %d's current floor mid-travel: B%d ...", elevator.getId(), elevator.getColumn().getId(), Math.abs(elevator.getCurrentFloor()));

    }
    //endregion
}
